hardware/raspberrypi/booteeprom.md

```diff
@@ -21,21 +21,25 @@ To check that the bootloader is working correctly, turn off the power, unplug ev
 
 If the Raspberry Pi is not booting it's possible that the bootloader EEPROM is corrupted. This can easily be reprogrammed using the Raspberry Pi Imager tool which is available via the [raspberrypi.org downloads page](https://www.raspberrypi.org/downloads/).
 
+Using the recovery image will erase any custom configuration options, resetting the bootloader back to factory defaults.
+
 ## Updating the bootloader
 
-We recommend setting up your Pi so that it automatically updates the bootloader: this means you will get new features and bug fixes as they are released. Bootloader updates are performed by the `rpi-eeprom` package, which installs a service that runs at boot-time to check for critical updates.
+Bootloader updates are instigated during a normal 'apt update`, `apt full-upgrade' cycle, this means you will get new features and bug fixes during your normal udpates. 
+
+Bootloader updates are performed by the `rpi-eeprom` package, which installs a service that runs at boot-time to check for critical updates. Although `rpi-eeprom` is available in the latest Raspberry Pi OS releases, there is usully no need to run it manually.
+
+To update your system, including the bootloader:
 
 ```
 sudo apt update
 sudo apt full-upgrade
```

Comment:
Not sure if we ought to add a `sudo reboot` to the end here? Also, if you're relying solely on `apt` to do the bootloader updates, I believe you actually need to (manually) reboot twice? The first reboot (after doing `sudo apt full-upgrade`) runs the systemd service which then spots that there's a newer bootloader update available and copies it to the boot partition, and the second reboot actually causes the bootloader to update itself (which then automatically reboots once it's finished) ? @timg236 ?

Reply:
I not made any changes based on the data here? Do we need any? The sequence described would, I think, be a useful addition to the boot sequence page https://www.raspberrypi.org/documentation/hardware/raspberrypi/bootmodes/bootflow_2711.md

Reply:
I think the sequence belongs elsewhere. It's a bit too involved for here.
I think the original comments was just that you didn't need to manually install rpi-eeprom on Raspberry Pi OS

Reply:
Sorry for the long ramble, I fell down a bit of a rabbit hole :rabbit2: :wink: 
My original question was whether `sudo reboot` should be added, and it has, so I'll mark this as resolved.